### ISSUE TITLE
Parametric values

### ISSUE THREAD

**Issue by marcocucinato** [thegouger/MIDI2LR#20]
A key or a CC could send to LR a calculated value, by using math expressions and variable substitutions.
for example, to put exposure at 50%

  <setting channel="1" controller="0" command="3" value="63"/>
or 
  <setting channel="1" controller="0" command="3;63"/>
 
To increment by 2 the Clarity for each key press
  <setting channel="1" controller="0" command="9;+2"/>
 
To give linear double slope to a control:
  <setting channel="1" controller="0" command="1;${0}*2"/>

**Comment by moutew** [comment]
This could be interesting to slow down tempareture settings also. The interval is so high ( 2000K to 50000K ) that I can't be very accurate as each step equals to an increment of 500k.

**Comment by PhantomLordM** [comment]
I agree with both of you guys. I wrote to @thegouger email with similar idea. I wonder if it would be possible to let user could select the step with which button will alter the value (+/- 1 or 2 or 5) and of course which value this button will change. 
However I think that issue #25 is addressing very similar trouble and from my point of view the better solution should be the one that is easier to implement and more universal.

**Comment by rsjaffe** [comment]
This issue was moved to rsjaffe/MIDI2LR#8
